src/lib/annotations/annotatorUtil.js

```diff
@@ -282,10 +309,7 @@ export function getAvatarHtml(avatarUrl, userId, userName) {
     let initials = '';
     if (userId !== '0') {
         // http://stackoverflow.com/questions/8133630/spliting-the-first-character-of-the-words
-        initials = userName
-            .replace(/\W*(\w)\w*/g, '$1')
-            .toUpperCase()
-            .substring(0, 3);
+        initials = userName.replace(/\W*(\w)\w*/g, '$1').toUpperCase().substring(0, 3);
```

Comment:
Just for record, we're planning on moving away from initials, and moving towards colours and symbols

Reply:
That was prettier, I didn't touch this line.